src/main/java/dev/j3fftw/litexpansion/ItemSetup.java

```diff
@@ -64,6 +63,7 @@ private void registerRubber() {
             registerNonPlaceableItem(Items.RUBBER, RubberSynthesizer.RECIPE_TYPE, SlimefunItems.OIL_BUCKET);
             new RubberSynthesizer().register(LiteXpansion.getInstance());
         }
+        rubberItem = SlimefunItem.getByID("RUBBER").getItem();
```

Comment:
This also doesn't fix the issue at all, you literally just moved your bug down a few lines...

Reply:
The bug occurred because it was trying to get the id of an item that didnt exist, right? By this point, if rubber is from slimytree taps, it will be fine (It was already before), and if the rubber is from LX, it wouldve been initialized inside the if.

Reply:
Yeah right